rest-api/edit-conversation-image-title/README.md

```diff
@@ -0,0 +1,17 @@
+This is an example project to go with our tutorial on How to edit the title and image of your TalkJS Conversation.
+<!-- Add link to tutorial once published -->
+
+The project uses TalkJS's [conversation actions](https://talkjs.com/docs/Features/Customizations/Conversation_Actions/) to add a custom action to the conversation that lets users edit the title or image of group conversations. The action is hooked to a backend server that calls the TalkJS REST API to update the conversation object.
+
+## Prerequisites
+
+To run this tutorial, you will need:
+
+- A [TalkJS account](https://talkjs.com/dashboard/login)
+- [Node.js](https://nodejs.org/en) and [npm](https://www.npmjs.com/)
+
+## How to run the tutorial
+
+1. Clone or download this project.
```

Comment:
This could also do with instructions on the following:
- how to fill in the app ID placeholder and env file
- how to add the 'editTitleOrImage' conversation action in the dashboard
(I realise these steps are also in the tutorial, but this way the code is self-contained and someone can set this up without reading everything)

Reply:
I have added the steps below. Let me know if its okay